app/models/students_group.rb

```diff
@@ -0,0 +1,4 @@
+class StudentsGroup < ApplicationRecord
+  belongs_to :student
+  has_many :groups
```

Comment:
belongs_to :groups
connecting tables have only belongs_to

Reply:
ok corrected that